harness/benchmarks/scripts/Stats.gd

```diff
@@ -1,66 +1,69 @@
+
 const SEC_IN_USEC = 1000000.0
 
 var data = []
 var sorted = false
 
 func add(time):
-    data.append(time)
+	data.append(time)
 
 func get_avg():
-    var total = 0.0
-    for i in data:
-        total += i
-    return total / len(data)
+	var total = 0.0
+	for i in data:
+		total += i
+	return total / len(data)
 
 func get_median():
-    var copy = data.duplicate()
-    copy.sort()
-    var length = len(copy)
-    var middle = length / 2
-    if length % 2 == 1:
-        return copy[middle]
-    else:
-        return (copy[middle - 1] + copy[middle]) / 2.0
+	var copy = data.duplicate()
+	copy.sort()
+	var length = len(copy)
+	var middle = length / 2
+	if length % 2 == 1:
+		return copy[middle]
+	else:
+		return (copy[middle - 1] + copy[middle]) / 2.0
 
 func get_percentile(percentile):
-    var copy = data.duplicate()
-    copy.sort()
-    var index = int(percentile * len(copy))
-    return copy[index]
+	var copy = data.duplicate()
+	copy.sort()
+	var index = int(percentile * len(copy))
+	return copy[index]
 
 func get_min():
-    var copy = data.duplicate()
-    copy.sort()
-    return copy[-1]
+	var copy = data.duplicate()
+	copy.sort()
+	return copy[-1]
 
 func get_max():
-    var copy = data.duplicate()
-    copy.sort()
-    return copy[0]
+	var copy = data.duplicate()
+	copy.sort()
+	return copy[0]
 
 func get_results():
-    var result = {}
-    result["raw"] = data.duplicate()
-    result["min"] = __to_op_per_s(get_min())
-    result["max"] = __to_op_per_s(get_max())
-    result["avg"] = __to_op_per_s(get_avg())
-    result["median"] = __to_op_per_s(get_median())
-    result["p95"] = __to_op_per_s(get_percentile(0.95))
-    result["p99"] = __to_op_per_s(get_percentile(0.95))
-    return result
+	var result = {}
+	result["raw"] = data.duplicate()
+	result["min"] = __to_op_per_s(get_min())
+	result["max"] = __to_op_per_s(get_max())
+	result["avg"] = __to_op_per_s(get_avg())
+	result["median"] = __to_op_per_s(get_median())
+	result["p05"] = __to_op_per_s(get_percentile(0.95))
+	result["p95"] = __to_op_per_s(get_percentile(0.05))
+	return result
 
 func __to_op_per_s(value):
-    if value == 0:
-        return value
-    return SEC_IN_USEC / value
+	if value == 0:
+		return value
+	return SEC_IN_USEC / value
 
 func _to_string():
-    var results = get_results()
-    var _min = results.min
-    var _max = results.max
-    var avg = results.avg
-    var median = results.median
-    var pc95 = results.p95
-    var pc99 = results.p99
-    var args = [_min, _max, avg, median, pc95, pc99]
-    return "min=%d, max=%d avg=%d op/s, median=%d op/s, 95pc=%d op/s, 99pc=%d op/s" % args
\ No newline at end of file
+	var results = get_results()
+	var _min = results.min
+	var _max = results.max
+	var avg = results.avg
+	var median = results.median
+	var pc05 = results.p05
+	var pc95 = results.p95
+	var args = [avg, _min, pc05, median, pc95, _max]
+	return "AVG=%d op/s, MIN=%d op/s, 05PC=%d op/s, MEDIAN=%d op/s, 95PC=%d op/s, MAX=%d op/s" % args
```

Comment:
could be use tabs between the results rather than spaces? so the numbers are evenly spaced for all benchmark methods? This would greatly improve readability IMO

Reply:
done

Reply:
![image](https://user-images.githubusercontent.com/31470327/101605860-fb931c80-3a02-11eb-8053-ff527f7d7c5a.png)
Using tabs gives us ugly results. As the tab push the next string to a multiple of 4 spaces so it doesn't align properly.

Reply:
Ok let's leave it then